footprint-benchmark/html/report/detail.html

```diff
@@ -0,0 +1,175 @@
+                <table class="table table-hover" style="width:100%; min-width: 800px;">
+                    <thead>
+                        <th style="width:20%">Process</th>
+                        <th style="width:20%">Max</th>
+                        <th style="width:20%">Min</th>
+                        <th style="width:20%">Avg</th>
+                        <th style="width:20%">Sum</th>
+                    </thead>
+                    <tbody id="measure_body" name="measure_body">
+                    </tbody>   
+                </table>
+            </div>
+            <div id="table-div" style="height: 400px;"></div>
+        </div>
+    </div>
+    
+    <script type="text/javascript">
+
+    $(document).ready(function(){
+        var measure = getQueryStringParams('measure');
+        loadDataJs(measure);
+
+        $("#headline").text("RackHD Benchmark Report - " + measure);
+
+        var case_data = case_info["log path"].split("-");
+        var atop_statistics = eval("atop_statistics_"+case_data[0]+"_"+case_data[1]);
+        var measure_data = eval(measure.toLowerCase() +
+                "_data_" +
+                case_data[0] +
+                "_"+case_data[1]);
+
+        $("#case_name").append(case_info["case name"]);
+        $("#case_log_path").append(case_info["log path"]);
+        $("#case_start_time").append(case_info["time marker"]["start"]);
+        $("#case_end_time").append(case_info["time marker"]["end"]);
+        $("#node_finish_time").append(case_info["time marker"]["node finish"]);
+
+        $("#table-header").text(measure + " Report");
+                
+        $.each(atop_statistics, function(i, val){
+            if(i=="CPU"){
+                $.each(val, function(process, value){
+                    var tr = document.createElement('tr');
+                    var td0 = document.createElement('td');
+                    td0.innerHTML = process;
+                    var td1 = document.createElement('td');
+                    td1.innerHTML = value["max"];
+                    var td2 = document.createElement('td');
+                    td2.innerHTML = value["min"];
+                    var td3 = document.createElement('td');
+                    td3.innerHTML = value["avg"].toFixed(2);
+                    var td4 = document.createElement('td');
+                    td4.innerHTML = "N/A";
+                    
+                    tr.appendChild(td0);
+                    tr.appendChild(td1);
+                    tr.appendChild(td2);
+                    tr.appendChild(td3);
+                    tr.appendChild(td4);
+                    
+                    $('#measure_body').append(tr);
+                })
+            }
+            
+        });
+
+    showGraph(measure_data,
+            getConfiguration(configuration, measure, 'Short Description'),
+            getConfiguration(configuration, measure, 'unit'));
+    });
+    
+    function zeropad(x) {

```

Comment:
Best practice: if not a global function, move to closure, for example: in `$(document).ready`.